src/puppetlabs/puppetdb/cli/time_shift_export.clj

```diff
@@ -76,13 +78,48 @@
       "facts" (time-shift-producer-timestamp json-data shift-period)
       "export-metadata" json-data)))
 
+(defn find-command-version
+  [entity]
+  (let[latest-command-versions command-constants/latest-command-versions]
+    (case entity
+      "facts" (:replace_facts latest-command-versions)
+      "catalogs"(:replace_catalog latest-command-versions)
+      "reports" (:store_report latest-command-versions))))
```

Comment:
This method is unnecessary because we are using fixed versions, available in function compare-command-versions.
We do this, so that if the implementation of a command changes, this script will fail, forcing us to make sure we do the correct data parsing.

Reply:
Updated to use the same values as compare-command-versions.